operations/template/alert.tf

```
@@ -120,3 +120,55 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     ]
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "ti-log-errors-alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-log-errors-alert"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+
+  action {
+    action_group  = [azurerm_monitor_action_group.notify_slack_email[count.index].id]
+    email_subject = "TI log errors detected!"
```

I believe the e-mail subject is easily seen in Slack.  To help differentiate between production and any of the other environments, I recommend putting the environment in the subject line.  Maybe something like...?

```suggestion
    email_subject = "${var.environment}: TI log errors detected!"
```